CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
 ### 2.0.0 (Next)
 
 * Your contribution here.
+* [#79](https://github.com/mongoid/mongoid-locker/pull/79): Update find_and_lock and find_and_unlock methods - [@dks17](https://github.com/dks17).
```

Comment:
Nitpick: these really improve performance as far as I can see, you may want to say "Improved performance of `find_and_lock` and `find_and_unlock`" instead.

Reply:
Accepted.